tiledb/sm/serialization/tiledb-rest.capnp

```diff
@@ -1261,6 +1261,19 @@ struct LoadArraySchemaResponse {
   # The loaded ArraySchema
 }
 
+struct QueryPlanRequest {
+  config @0 :Config;
+  # Config
+
+  query @1 : Query;
+  # the query for which we request the plan
+}
+
+struct QueryPlanResponse {
+  queryPlan @0 :Text;
```

Comment:
I wonder if it would be a good idea to have the REST API just return a JSON string without wrapping it inside capnproto.

Reply:
It's standard practice to use `capnproto` for our REST API so that we can leverage backwards compatibility if we need to update the messages in the future to include more information.

Reply:
Capnproto support both binary and json for over the wire as we use it. All serialization work must use cap'n proto, and since we care about performance we want to default to cap'n proto binary serialization in all cases.